corehq/apps/registration/utils.py

```diff
@@ -282,3 +286,20 @@ def create_30_day_trial(domain_obj):
     )
     trial_subscription.is_active = True
     trial_subscription.save()
+
+
+def create_free_community_plan(domain_obj):
+    community_plan_version = DefaultProductPlan.get_default_plan_by_domain(
+        domain_obj, edition=SoftwarePlanEdition.COMMUNITY
+    )
+    community_account = BillingAccount.objects.get_or_create(
+        name="Free Community Account for %s" % domain_obj.name,
```

Comment:
I think this function doesn't have to be used, since community plans are represented by not having an active subscription.  The billing account will be initialized if needed for a prepayment or invoicing.  Sorry this was not clear in the spec.